pkg/log/status.go

```diff
@@ -26,9 +26,11 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"runtime"
 	"strings"
 
 	"github.com/fatih/color"
```

Comment:
Can we remove this lib, then?

Reply:
We still need this for Linux color support.